docs/source/optim.rst

```diff
@@ -130,7 +130,7 @@ Algorithms
 .. autoclass:: SGD
     :members:
 
-How to adjust Learning Rate
+How to adjust learning rate
```

Comment:
Why? Seems other section titles also do not use capital letters except the first letter

Reply:
Which is exactly why I did this...

Reply:
Right... I saw it backwards XD